tests/cmdline/commands/test_process.py

```diff
@@ -376,74 +382,54 @@ def test_multiple_processes(self):
         assert str(self.node_root.pk) in get_result_lines(result)[2]
 
 
-@pytest.mark.skip(reason='fails to complete randomly (see issue #4731)')
 @pytest.mark.requires_rmq
-@pytest.mark.usefixtures('with_daemon', 'aiida_profile_clean')
-@pytest.mark.parametrize('cmd_try_all', (True, False))
-def test_pause_play_kill(cmd_try_all, run_cli_command):
-    """
-    Test the pause/play/kill commands
-    """
-    # pylint: disable=no-member, too-many-locals
-    from aiida.cmdline.commands.cmd_process import process_kill, process_pause, process_play
-    from aiida.engine import ProcessState
-    from aiida.manage import get_manager
-    from aiida.orm import load_node
+@pytest.mark.usefixtures('started_daemon_client')
+def test_process_pause(submit_and_await, run_cli_command):
+    """Test the ``verdi process pause`` command."""
+    node = submit_and_await(WaitProcess)
```

Comment:
No, you are right. By default the process should not be paused so that can be simply an assert. It is to make sure that in the next step if the check `node.paused == True` passes, it is due to the command, and not because it was paused to begin with.